sdk/servicebus/service-bus/src/serializers/queueResourceSerializer.ts

```diff
@@ -40,15 +47,25 @@ export function buildQueueOptions(queue: CreateQueueOptions): InternalQueueOptio
     DuplicateDetectionHistoryTimeWindow: queue.duplicateDetectionHistoryTimeWindow,
     MaxDeliveryCount: getStringOrUndefined(queue.maxDeliveryCount),
     EnableBatchedOperations: getStringOrUndefined(queue.enableBatchedOperations),
-    AuthorizationRules: getRawAuthorizationRules(queue.authorizationRules),
+    
+    // TODO: found while syncing the ordering. This property is in .net, but not in ours?
+    //IsAnonymousAccessible: false,
```

Comment:
Service team recommended removing it as it is a "legacy" option not applicable anymore

Reply:
comment removed.